docs/ko/docs/tutorial/dependencies/index.md

```diff
@@ -0,0 +1,233 @@
+# 의존성 - 도입부
+
+**FastAPI**는 강력하면서도 직관적인 **<abbr title="also known as components, resources, providers, services, injectables">의존성 주입</abbr>** 시스템입니다.
+
+**FastAPI**는 사용하기 매우 쉽게 디자인되었으며, 어느 개발자라도 다른 컴포넌트들과 통합하기 쉽게 만들어졌습니다.
+
+## "의존성 주입"이 무엇입니까?
+
+**"의존성 주입"**은 프로그래밍에서 코드(이 경우, 당신의 *경로 작동 함수*)에 작동하고 사용하는데 필요한 "의존성"들이 선언하는 방법입니다.
+
+그리고, 그 시스템(**FastAPI**의 경우)은 필요한 의존성을 코드에 제공하는 데 필요한 모든 작업을 처리합니다(의존성 "주입").
+
+다음의 경우에 매우 유용합니다:
+
+* 공유 로직이 존재 (같은 코드 로직의 반복).
+* 데이터베이스 연결 공유.
+* 보안, 인증, 권한 등의 강화.
+* 그 외 여러가지...
+
+이 모든 것이 코드의 반복을 최소화를 위함입니다.
+
+## 첫 번째 단계
+
+매우 간단한 예시를 봅시다. 간단하지만 유용하지는 않죠, 아직은 말입니다.
+
+그러나 이 방법은 어떻게 **의존성 주입** 시스템이 작동하는지를 보여줍니다.
+
+### 의존성 또는 "의존적인" 생성
+
+먼저 의존성에 초점을 맞춰봅시다.
+
+그것은 단지 *경로 작동 함수*가 취할 수 있는 것과 동일한 모든 매개변수를 취할 수 있는 함수일 뿐입니다:
+
+=== "Python 3.7 이상"
+
+    ```Python hl_lines="8-11"
+    {!> ../../../docs_src/dependencies/tutorial001.py!}
+    ```
+
+=== "Python 3.10 이상"
+
+    ```Python hl_lines="6-7"
+    {!> ../../../docs_src/dependencies/tutorial001_py310.py!}
+    ```
+
+끝입니다.
+
+**2 lines**.
+
+그리고 그것은 모든 *경로 작동 함수*이 가지고 있는 것과 같은 모양과 구조를 가지고 있습니다.
+
+"데코레이터" 없이(`@app.get("/some-path")` 없이) *경로 작동 함수*로 생각할 수 있습니다.
+
+그리고 원한다면 언제든지 되돌릴 수 있습니다.
```

Comment:
Original sentence:
> And it can return anything you want.

I don't think the meaning of `return` is to `restore` on here. It should be Return statement in programming.
So this is my suggestion:

```suggestion
그리고 원하는 타입대로 반환할 수도 있습니다.
```